### PR TITLE
ECDb Profile Version Update 4.0.0.5

### PR DESCRIPTION
**Pull request by rschili**
Profile update which includes
- new ECDBMap
- new ECDbMeta

itwinjs-core: https://github.com/iTwin/itwinjs-core/pull/6600

### CODE REVIEW

File: iModelCore/ECDb/Tests/NonPublished/ClassViewsTests.cpp [hunk 1]
```diff
@@ -1801,4 +1770,346 @@ TEST_F(ClassViewsFixture, update_views_in_dynamic_schema_cte) {
...
+    //ASSERT_EQ(true, cidColInfo.IsSystemProperty()); Does not work yet! Returns false.
+    ASSERT_TRUE(classId.IsValid());
+    ASSERT_EQ(fruitViewClassId, classId);
+
+    ASSERT_STREQ("Banana", stmt.GetValueText(2));
+    auto nameColInfo = stmt.GetColumnInfo(2);
+    //ASSERT_EQ(false, nameColInfo.IsGeneratedProperty()); These also do not work yet for views!
+    //ASSERT_STREQ(nameColInfo.GetRootClass().GetClass().GetName().c_str(), "FruitView");
+    //ASSERT_STREQ(nameColInfo.GetProperty()->GetClass().GetName().c_str(), "FruitView");
+    ASSERT_EQ(BE_SQLITE_DONE, stmt.Step());
```
Comment (ColinKerr) [inline iModelCore/ECDb/Tests/NonPublished/ClassViewsTests.cpp:1844]: do these commented out asserts still not work or is this something you forgot to uncomment?

**Review by khanaffan — APPROVED** [review]

**Review by grigasp — APPROVED** [review]
approving the ECPresentation change

**Comment by rschili** [comment]
Just a reminder comment (close when done)
Create and link an itwinjs-core PR with these changes:
- Update nextversion.md
- Fix one test with a hard coded classid, it is currently failing. This one:
```
 1) Changeset Reader API
       Changeset reader / EC adaptor:

      AssertionError: expected { Id: '0x20000000002', …(1) } to deeply equal { Id: '0x20000000002', …(1) }
      + expected - actual
  ...2 lines omitted...
         "Id": "0x20000000002"
      -  "RelECClassId": "0x6d"
      +  "RelECClassId": "0x60"
       }

      at Context.<anonymous> (src\test\standalone\ChangesetReader.test.ts:151:14)
```

**Review by pmconne — APPROVED** [review]

**Pull request merged**